net/lp_rpc.proto

```diff
@@ -14,7 +14,7 @@ service Transcoder {
 
   // Called by the transcoder to register to an orchestrator. The orchestrator
   // notifies registered transcoders of segments as they come in.
-  rpc RegisterTranscoder(RegisterRequest) returns (stream NotifySegment);
+  rpc RegisterTranscoder(stream RegisterRequest) returns (stream NotifySegment);
```

Comment:
Curious, what's the advantage of making the RegisterTranscoder request a stream, rather than just having an explicit UnregisterTranscoder RPC call?

Reply:
It allows to also to catch situation when connection was lost but notification for it was not sent.

Reply:
The approach of turning the Register request into a stream feels weird. Will cancelling the context that we pass into `RegisterTranscoder` work? We'd probably have to propagate it from `RunTranscder / runTranscoder` . See https://github.com/grpc/grpc-go/issues/990